ads/jobs/builders/infrastructure/dsc_job.py

```diff
@@ -833,6 +841,13 @@ class DataScienceJob(Infrastructure):
             .with_shape_config_details(memory_in_gbs=16, ocpus=1)
             # Minimum/Default block storage size is 50 (GB).
             .with_block_storage_size(50)
+            # A list of file systems to be mounted
+            .with_storage_mount(
+                {
+                    "src" : "<mount_target_ip_address>:<export_path>",
+	                "dest" : "<destination_directory_name>"
```

Comment:
Align "src" and "dest"
```suggestion
	            "dest" : "<destination_directory_name>"
```